CITATION.cff

```diff
@@ -0,0 +1,86 @@
+cff-version: 1.2.0
+title: A Python Library for NeXus-Compliant Experimental Data Conversion and Integration with NOMAD Platform
+message:
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+version: 0.7.2
+authors:
+  - given-names: Sherjeel
+    family-names: Shabih
+    orcid: 'https://orcid.org/0009-0008-6635-4465'
+  - given-names: Florian
+    family-names: Dobener
+    orcid: 'https://orcid.org/0000-0003-1987-6224'
+  - given-names: Markus
+    family-names: Kühbach
+    orcid: 'https://orcid.org/0000-0002-7117-5196'
+  - given-names: Andrea
+    family-names: Albino
+    orcid: 'https://orcid.org/0000-0001-9280-7431'
+  - given-names: Sandor
+    family-names: Brockhauser
+    orcid: 'https://orcid.org/0000-0002-9700-4803'
+  - given-names: Sebastian
+    family-names: Brückner
+    orcid: 'https://orcid.org/0000-0002-5969-847X'
+  - given-names: Theodore
+    family-names: Chang
+    orcid: 'https://orcid.org/0000-0002-4911-0230'
+  - given-names: Ádám
+    family-names: Fekete
+    orcid: 'https://orcid.org/0000-0002-6263-897X'
+  - given-names: Lev
+    family-names: Ginzburg
+    orcid: 'https://orcid.org/0000-0001-8929-1040'
+  - given-names: Tamás
+    family-names: Haraszti
+    orcid: 'https://orcid.org/0000-0002-7095-4358'
+  - given-names: Ron
+    family-names: Hildebrandt
+    orcid: https://orcid.org/0000-0001-6932-604X
+  - given-names: Sabine
+    family-names: Maier
+    orcid: 'https://orcid.org/0000-0001-9589-6855'
+  - given-names: José A.
+    family-names: Márquez
+    orcid: 'https://orcid.org/0000-0002-8173-2566'
+  - given-names: Rubel
+    family-names: Mozumder
+    orcid: 'https://orcid.org/0009-0007-5926-6646'
+  - given-names: Lukas
+    family-names: Pielsticker
+    orcid: 'https://orcid.org/0000-0001-9361-8333'
+  - given-names: Laurenz
+    family-names: Rettig
+    orcid: 'https://orcid.org/0000-0002-0725-6696'
+  - given-names: Heiko B.
+    family-names: Weber
+    orcid: 'https://orcid.org/0000-0002-6403-9022'
+
+# doi: Not available yet
+repository-code: 'https://github.com/FAIRmat-NFDI/pynxtools'
+# url: Not available yet
+abstract:
+
+  pynxools is a python library that provides core functionalities for converting experimental
+  data, if the experiment specific reader plugin (e.g. pynxtools-stm for Scanning Tunneling
+  Microscopy, pynxtools-em for for Electron Microscopy) is integrated with, into NeXus-compliant
+  HDF5 file following NeXus application definition
+  (https://manual.nexusformat.org/classes/applications/index.html). During the data
```

Comment:
```suggestion
  pynxtools is a Python package that provides core functionalities for converting experimental data into HDF5 files that are compliant with NeXus application definitions (https://manual.nexusformat.org/classes/applications/index.html). During the data
```
I would strongly suggest to not mention the reader plugins here (these also change over time).